### ISSUE TITLE
no such file to load -- logstash-input-kinesis_jars

### ISSUE THREAD

**Issue by Gauravshah** [codekitchen/logstash-input-kinesis#1]
tried using this gem but getting `no such file to load -- logstash-input-kinesis_jars` . 

 `logstash-input-kinesis_jars` is added to https://github.com/codekitchen/logstash-input-kinesis/blob/master/.gitignore and hence it is not working

**Comment by codekitchen** [comment]
The jars are intentionally not included in the source, they're downloaded during gem install. This should be handled by logstash's `plugin install` command, as outlined in the README. How did you install the plugin? What version of logstash?

**Comment by Gauravshah** [comment]
I am trying it on logstash 1.4.2 

``` bash

GSMBP:logstash-1.4.2 gaurav$ env GEM_HOME=vendor/bundle/jruby/1.9 GEM_PATH="" java -jar vendor/jar/jruby-complete-1.7.11.jar -S gem install logstash-input-kinesis
Fetching: jar-dependencies-0.1.7.gem (100%)
Successfully installed jar-dependencies-0.1.7
Fetching: thread_safe-0.3.5-java.gem (100%)
Successfully installed thread_safe-0.3.5-java
Fetching: descendants_tracker-0.0.4.gem (100%)
Successfully installed descendants_tracker-0.0.4
Fetching: equalizer-0.0.11.gem (100%)
Successfully installed equalizer-0.0.11
Fetching: coercible-1.0.0.gem (100%)
Successfully installed coercible-1.0.0
Fetching: ice_nine-0.11.1.gem (100%)
Successfully installed ice_nine-0.11.1
Fetching: axiom-types-0.1.1.gem (100%)
Successfully installed axiom-types-0.1.1
Fetching: virtus-1.0.5.gem (100%)
Successfully installed virtus-1.0.5
Fetching: maven-tools-1.0.7.gem (100%)
Successfully installed maven-tools-1.0.7
Fetching: ruby-maven-libs-3.1.1.gem (100%)
Successfully installed ruby-maven-libs-3.1.1
Fetching: ruby-maven-3.1.1.0.8.gem (100%)
Successfully installed ruby-maven-3.1.1.0.8
Fetching: logstash-input-kinesis-1.2.2-java.gem (100%)
Successfully installed logstash-input-kinesis-1.2.2-java
12 gems installed
```

**Comment by codekitchen** [comment]
Ah, as stated in the README, this plugin requires Logstash 1.5 (which is still in beta).

**Comment by mdiscenza** [comment]
Hi, I am having a similar issue. I've tried this with logstash versions 1.5.1 and 1.5.0 and keep getting 

```
The error reported is: 
Couldn't find any input plugin named 'kinesis'. Are you sure this is correct? Trying to load the kinesis input plugin resulted in this error: no such file to load -- logstash-input-kinesis_jars
```

This is after I ran `/opt/logstash/bin/plugin install logstash-input-kinesis` and was notified that the install was successful.  I, however, cannot find 'logstash-input-kinesis_jars' - this would be a file in the 'intputs' directory?

**Comment by codekitchen** [comment]
hm I'm still on logstash version 1.5.0.rc4, it's possible something changed in the final release. I'll try it out soon.

**Comment by codekitchen** [comment]
I'm able to reproduce the error in logstash 1.5.1, but not 1.5.0.

I noticed during the plugin install on 1.5.1 that I actually got a bundler error, though it then acted as if the plugin install was successful anyway:

```
Validating logstash-input-kinesis-1.2.4
Installing logstash-input-kinesis
Error Bundler::InstallError, retrying 1/10
An error occurred while installing logstash-input-kinesis (1.2.4), and Bundler cannot continue.
Make sure that `gem install logstash-input-kinesis -v '1.2.4'` succeeds before bundling.
Installation successful
```

I'll figure out a fix.

**Comment by codekitchen** [comment]
This is fixed in version 1.3.1, which now requires logstash 1.5.1 -- they changed the way that jar dependencies are handled.
